From 351069fbe42568dceb7df560264d48dd2556a5cb Mon Sep 17 00:00:00 2001
From: Isaac Chen <isaacchen@isaacchen.cn>
Date: Wed, 6 Jun 2018 20:43:17 +0800
Subject: [PATCH 168/249] wiki: build: Remove libesd0-dev

 * It is not exist on Ubuntu 18.04(bionic) and without it build is also successful,so remove it

Change-Id: I629a35858c3dfc420c059a6ea1a2ad4b3f182596
Signed-off-by: Isaac Chen <isaacchen@isaacchen.cn>
---
 _includes/templates/device_build.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_includes/templates/device_build.md b/_includes/templates/device_build.md
index d70697f..1af85a7 100644
--- a/_includes/templates/device_build.md
+++ b/_includes/templates/device_build.md
@@ -78,7 +78,7 @@ command directly in the Terminal." %}
 To build LineageOS, you'll need:
 
 * `bc bison build-essential ccache curl flex g++-multilib gcc-multilib git gnupg gperf imagemagick
-   lib32ncurses5-dev lib32readline-dev lib32z1-dev libesd0-dev liblz4-tool libncurses5-dev
+   lib32ncurses5-dev lib32readline-dev lib32z1-dev liblz4-tool libncurses5-dev
    libsdl1.2-dev libssl-dev libwxgtk3.0-dev libxml2 libxml2-utils lzop pngcrush rsync
    schedtool squashfs-tools xsltproc {% if cpu_architecture contains 'x86' %}yasm {% endif %}
    zip zlib1g-dev`
-- 
2.17.1

